server/cliserver_test.go

```diff
@@ -34,6 +35,6 @@ func TestGetStatus(t *testing.T) {
 	defer res.Body.Close()
 	body, err := ioutil.ReadAll(res.Body)
 	req.Nil(err)
-	assert.Equal(`{"Manifests":{},"OrchestratorPool":[],"Version":"undefined","RegisteredTranscodersNumber":1,"RegisteredTranscoders":[{"Address":"TestAddress","Capacity":5}],"LocalTranscoding":false}`,
+	assert.Equal(`{"Manifests":{},"OrchestratorPool":[],"Version":"undefined","CompilerVersion":"`+runtime.Version()+`","RegisteredTranscodersNumber":1,"RegisteredTranscoders":[{"Address":"TestAddress","Capacity":5}],"LocalTranscoding":false}`,
```

Comment:
I would suggest using `fmt.Sprintf` here rather than the nested backticks.  It would expand to more lines of code, but might be easier to read and edit later on.

Maybe this should be called `RuntimeVersion` instead of `CompilerVersion`?

Golang uses the `gcc` compiler, and it might be any version.  I'm not sure if you're looking to track that, or if its easily discoverable.

What is this endpoint used for?  Do you think we should also include `sys.GOARCH` and `sys.GOOS` for architecture target and running OS?

Reply:
> I would suggest using fmt.Sprintf 

fixed

> I think this should be called RuntimeVersion instead of CompilerVersion

renamed to `GORuntimeVersion`

> Golang uses the gcc compiler, and it might be any version. I'm not sure if you're looking to track that, or if its easily discoverable.

> Golang uses the gcc compiler, and it might be any version. I'm not sure if you're looking to track that, or if its easily discoverable.

`runtime.Compiler` returns just `gc` - not very useful. 

> Do you think we should also include sys.GOARCH and sys.GOOS for architecture target and running OS?

Not sure if Livepeer node will every be running on something other than `amd64` (on phones? RaspberryPi?), but to be future proof, added GOARCH and GOOS to output.

There was two reasons to adding all this:
1. For future bug-reports from users.
2. For own testing - in test harness I often use image built in Docker Hub, and I want to check with what version of `golang` it was built - we already once hit bug that was manifested itself only in `go1.11`, but not in `go1.10` and not in `go1.12`.